src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> head;
+    private Node<T> tail;
+    private int size;
+
     @Override
     public void add(T value) {
+        Node<T> newNode = new Node<>(value, tail, null);
```

Comment:
When creating a new `Node<T>`, pass all the fields to the `Node` constructor. It's better to pass 'null' explicitly for the 'next' parameter here for clarity.